test/integration/evictions/evictions_test.go

```diff
@@ -334,6 +339,86 @@ func TestEvictionVersions(t *testing.T) {
 	}
 }
 
+// func TestEvictionWithJobBackoffPolicy tests the behaviour when the JobBackoffPolicy feature is enabled
+func TestEvictionWithJobBackoffPolicy(t *testing.T) {
```

Comment:
```suggestion
func TestEvictionWithFinalizers(t *testing.T) {
```

Eventually the feature gate will go away, but we want to keep this test.